compoments/textInput.js

```diff
@@ -63,31 +62,38 @@ class TextInput extends Component {
     })
   };
 
-  render(){
+  render() {
     return (
       <>
-        <h1 className="govuk-heading-l">{this.title}</h1>
-        <div className={this.state.error.msg ? 'govuk-form-group--error' : 'govuk-form-group'}>
-          <form action="" className={(this.long ? 'govuk-grid-column-two-thirds':'govuk-grid-column-one-third')+' govuk-!-padding-0'}>
+        <div
+          className={this.state.error.msg ? 'govuk-form-group--error' : 'govuk-form-group'}>
+          <form action=""
+            className={(this.long ? 'govuk-grid-column-two-thirds' : 'govuk-grid-column-one-third') + ' govuk-!-padding-0'}>
```

Comment:
I'm trying to understand the `this.long` ternary operator? Is it to define the width of the text input? 

This should be done on the input with [fixed-width classes](https://design-system.service.gov.uk/components/text-input/#fixed-width-inputs)

Reply:
It's a boolean which is passed to the TextInput component, so this conditionally sets the class on this form. Currently, it is false on the postcode page and true on the contact person page, so the form should render with govuk-grid-column-one-third on the postcode page.

This css was already present and these are just formatting/spacing changes to the code. I think changes to this input (which also affects the contact-person page) is separate work.